### PR TITLE
Preparing for new release

### PR DESCRIPTION
**Pull request by andriyun**

### CODE REVIEW

File: os2forms_forloeb.module [hunk 1]
```diff
@@ -268,3 +270,59 @@ function os2forms_forloeb_form_alter(&$form, FormStateInterface $form_state, $fo
...
+ * Allows requests with tokens to view the entity.
+ */
+function os2forms_forloeb_entity_access(\Drupal\Core\Entity\EntityInterface $entity, $operation, \Drupal\Core\Session\AccountInterface $account) {
+  if ($operation == 'update' && $entity instanceof WebformSubmission) {
+    $token = \Drupal::request()->query->get('os2forms-forloeb-ws-token');
+    if ($token && $token === $entity->getToken()) {
+      return \Drupal\Core\Access\AccessResult::allowed();
+    }
+  }
+}
```
Comment (rimi-itk) [inline os2forms_forloeb.module:315]: Implementations of `hook_entity_access` should return an instance of `AccessResultInterface` (cf. https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21entity.api.php/function/hook_entity_access/9.1.x), so I think we should return `AccessResult::neutral()` here.

File: os2forms_forloeb.module [hunk 2]
```diff
@@ -268,3 +270,59 @@ function os2forms_forloeb_form_alter(&$form, FormStateInterface $form_state, $fo
...
+
+/**
+ * Implements hook_entity_access().
+ * Allows requests with tokens to view the entity.
+ */
+function os2forms_forloeb_entity_access(\Drupal\Core\Entity\EntityInterface $entity, $operation, \Drupal\Core\Session\AccountInterface $account) {
+  if ($operation == 'update' && $entity instanceof WebformSubmission) {
+    $token = \Drupal::request()->query->get('os2forms-forloeb-ws-token');
+    if ($token && $token === $entity->getToken()) {
+      return \Drupal\Core\Access\AccessResult::allowed();
```
Comment (rimi-itk) [inline os2forms_forloeb.module:312]: Add `use Drupal\Core\Access\AccessResult` rather than using the FQN here.

**Review by rimi-itk — CHANGES_REQUESTED** [review]
A few minor changes needed.

File: os2forms_forloeb.module (hunk 2 above)
Comment (madsnorgaard) [reply os2forms_forloeb.module:312]: Haven't tested. LGTM.

**Review by madsnorgaard — COMMENTED** [review]
LGTM. No tests done during review.

File: os2forms_forloeb.module (hunk 1 above)
Comment (andriyun) [reply os2forms_forloeb.module:315]: Added this fix in #PR133

File: os2forms_forloeb.module (hunk 2 above)
Comment (andriyun) [reply os2forms_forloeb.module:312]: Added this fix in https://github.com/OS2Forms/os2forms_forloeb/pull/133

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**